Gemfile.lock

```diff
@@ -8,20 +8,18 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (4.2.6)
+    activesupport (5.0.1)
```

Comment:
Oh.... This could be a problem....

If we use activesupport 4.x, then this relies on json `1.x`, which displays deprecation warnings for ruby `2.4`.

On the other hand, if we use activesupport `5.0.x`, then this is requires ruby version `>= 2.2.2`.

Unless a "fixed" version of the json `1.x` gem is released, I don't think we can resolve the deprecation warning without also dropping support for ruby `<= 2.2.1`.